Activities/GAScripted.cpp

```diff
@@ -45,8 +45,7 @@ ConcreteClassInfo(GAScripted, GameActivity, 0);
 // Description:     Clears all the member variables of this GAScripted, effectively
 //                  resetting the members of this abstraction level only.
 
```

Comment:
This file has lots of changes, but it's entirely just cleanup, no changes in behaviour here.